packages/inter-protocol/src/reserve/assetReserve.js

```diff
@@ -66,11 +66,11 @@ export const prepare = async (zcf, privateArgs, baggage) => {
   };
   trace('awaiting takeFeeMint');
   const feeMint = await takeFeeMint();
+  const storageNode = await privateArgs.storageNode;
```

Comment:
I think this is an improvement regardless, so no change suggested. But I do want to double check my understanding. Given the `await` already present on line 68, the original code was safe by our new `await` rules, and this is only a readability improvement, yes?